docker/Dockerfile

```diff
@@ -1,24 +1,27 @@
 ARG VERSION
 
-FROM ghcr.io/deephaven/server:${VERSION}
+FROM ghcr.io/deephaven/server-sklearn:${VERSION}
 
 ARG DH_IB_VERSION
 ARG DH_IB_BRANCH
 
+COPY deephaven_ib-0.2.4-py3-none-any.whl /tmp/
+
 # Install requirements
 RUN apt update && \
-    apt install --yes git python3-venv && \
-    python3 -m pip install --upgrade pip && \
-    python3 -m pip install --upgrade setuptools wheel build twine
-
-# Invalidate the cache below here, so that the latest repo changes get cloned
-ARG CACHEBUST
+    apt install --yes git python3-venv zip && \
+    python3 -m pip install --upgrade pip
 
-# Build and install deephaven-ib
 RUN mkdir /build && \
     cd /build && \
-    git clone --branch ${DH_IB_BRANCH} https://github.com/deephaven-examples/deephaven-ib.git && \
-    cd deephaven-ib && \
-    python3 -m build && \
-    pip3 install dist/*.whl && \
-    rm -rf /build
+    curl -o ./api.zip "https://interactivebrokers.github.io/downloads/twsapi_macunix.1016.01.zip" && \ 
+    unzip api.zip && \
+    cd ./IBJts/source/pythonclient && \
+    python3 setup.py install && \
+    rm -rf ./build
```

Comment:
Make sure that rm is cleaning up all "junk", so that the docker image isn't bloated.